test/Microsoft.TemplateEngine.Cli.UnitTests/CacheUpgradeTests.cs

```diff
@@ -14,16 +15,23 @@ public void CanReadUnversionedCache()
             IEngineEnvironmentSettings mockEnvironmentSettings = new MockEngineEnvironmentSettings();
             TemplateCache cache = new TemplateCache(mockEnvironmentSettings, CacheDataOriginalStyle);
 
-            Assert.Equal(3, cache.TemplateInfo.Count);
+            Assert.NotEqual(TemplateCache.CurrentVersion, cache.Version);
+            Assert.Equal(0, cache.TemplateInfo.Count);
         }
 
         [Fact(DisplayName = nameof(CanReadVersion1000Cache))]
         public void CanReadVersion1000Cache()
         {
-            IEngineEnvironmentSettings mockEnvironmentSettings = new MockEngineEnvironmentSettings();
+            MockEngineEnvironmentSettings mockEnvironmentSettings = new MockEngineEnvironmentSettings();
+            mockEnvironmentSettings.Host= new TestHost
+            {
+                HostIdentifier = "TestRunner",
+                Version = "1.0.0.0",
```

Comment:
I question the need to specify a version number when creating a host now. The host already has one cache serialization implementation inside anyway and giving a different number won't change anything. Should we put the version number in a deeper level of abstraction (for instance, as a private static member into the class that handle serialization)?